README.md

```diff
@@ -358,7 +363,7 @@ Device types like `bridge`, `bond`, `team` work similar:
 network_connections:
   - name: "br0"
```

Comment:
If quotes were removed around eth0 above, they should be here as well for consistency.

Reply:
I removed the quotes from other appearances as well.